src/Store.java

```diff
@@ -0,0 +1,168 @@
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class Store {
+
+    static Bill oneUah = new Bill(1, 0);
```

Comment:
I think these are the constant values references to which won't change. Please declare them as true constants
```
static final Bill ONE_UAH = new Bill(1,0);
```